Makefile

```diff
@@ -195,76 +191,6 @@ image-build-iso: rpm
 iso: image-build-configure image-build-iso
 .PHONY: iso
 
-###############################
```

Comment:
When removing the containerized builds, you'll also want to remove their systemd units (`packaging/systemd/microshift-aio.service` and `packaging/systemd/microshift-containerized.service`) and drop them from the RPM package.